feat(docs): tls document improvement (#373)
emqx/cloud-docs: en_US/deployments/stop_delete_deployment.md

# Stop and Delete Deployments

## Serverless

### Stop a Deployment

> 1. Go to the **Cloud Console** and click on the deployment you wan t to **stop** to access the deployment overview page.

Reviewer: ```suggestion
1. Go to the **Cloud Console**. Click on the deployment you want to **stop** to access the deployment overview page.
```

2. Click the **Stop** button and enter the name of the deployment in the pop-up box.

> 3. Click **Confirm** to stop the deployment .

Reviewer: ```suggestion
3. Click **Confirm** to stop the deployment.
```

> ::: warning

Reviewer: ```suggestion
::: Warning
```

Stopping a deployment will disconnect devices from the deployment, but your data and connection address will be preserved.
:::

### Delete a Deployment

> 1. Go to the **Cloud Console** and click on the deployment you wan t to **delete** to access the deployment overview page.

Reviewer: ```suggestion
1. Go to the **Cloud Console**. Click on the deployment you want to **delete** to access the deployment overview page.
```

Reviewer: Changes also apply to the following sections.

Deleting a deployment will disconnect devices from the deployment and remove all deployment data and configurations.
:::

## Dedicated

### Stop a Deployment

> ::: warning

Reviewer: ```suggestion
::: Warning
```

3. Click **Confirm** to stop the deployment .

### Delete a Deployment

> ::: warning

Reviewer: ```suggestion
::: Warning
```

2. Click the **Delete** button and enter the name of the deployment in the pop-up box.

> 3. Click **Confirm** to stop the deployment .

Reviewer: ```suggestion
3. Click **Confirm** to stop the deployment.
```
